index.js

```diff
@@ -1,36 +1,53 @@
 var express = require('express');
 var bodyParser = require('body-parser');
+var DataStore = require('nedb');
 
 var port = 3000;
 var BASE_API_PATH = "/api/v1";
-
-var contacts = [
-    {"name" : "peter", "phone": "12345"},
-    {"name" : "jhon", "phone" :"6666"}
-];
+var DB_FILE_NAME= __dirname + "/contacts.json";
 
 console.log("starting API server...");
 
 var app = express();
 app.use(bodyParser.json());
 
+var db= new DataStore({
+    filename: DB_FILE_NAME,
+    autoload: true
+});
+
 app.get("/", (req, res) => {
     res.send("<html><body><h1>My server</h1></body></html>");
 });
 
 app.get(BASE_API_PATH + "/contacts", (req, res) => {
     console.log(Date() + " - GET /contacts");
-    res.send(contacts);
+    db.find({},(err, contacs) =>{
+        if(err){
+            console.log(Date() + "-" + err);
+            res.sendStatus(500);
+        } else {
+            res.send(contacs);
```

Comment:
Please, change this code to remove the _id from the information  that is sent to the user

Reply:
change done